From 4c5552b0c46e0005abae7ed03d2944b03338d3bb Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Thu, 09 Jan 2020 15:45:36 +0200
Subject: [PATCH] [SW-6739] arcboot: adjust key size for rsa_fixed

rsa_fixed key size is 8 bytes bigger, adding a fixed
exponent for faster calculation algorithem

Change-Id: I8cf8a5a6dc42fbd195bbda6c17b684d54937da97
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/include/arcboot.h b/include/arcboot.h
index 8320116..c952692 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -13,7 +13,7 @@
 
 #define FW_SIZE					65535
 #define SIG_SIZE				384
-#define KEY_SIZE				384
+#define KEY_SIZE				392
 #define BLOB_START				4
 
 #define SPI_FLASH_BLOCK_SIZE			1024
